github/resource_github_organization_custom_properties.go

```diff
@@ -79,14 +85,22 @@ func resourceGithubCustomPropertiesCreate(d *schema.ResourceData, meta any) erro
 		allowedValuesString = append(allowedValuesString, v.(string))
 	}
 
-	customProperty, _, err := client.Organizations.CreateOrUpdateCustomProperty(ctx, ownerName, d.Get("property_name").(string), &github.CustomProperty{
+	customProperty := &github.CustomProperty{
 		PropertyName:  &propertyName,
 		ValueType:     valueType,
 		Required:      &required,
 		DefaultValue:  &defaultValue,
 		Description:   &description,
 		AllowedValues: allowedValuesString,
-	})
+	}
+
+	// Set ValuesEditableBy if provided
+	if valuesEditableBy, ok := d.GetOk("values_editable_by"); ok {
+		valuesEditableByStr := valuesEditableBy.(string)
+		customProperty.ValuesEditableBy = &valuesEditableByStr
+	}
```

Comment:
question: What caused you to choose to follow a different pattern than what is used for the other optional properties?